### PR TITLE
[Chore] Migrate to Xcode 11.3

### PR DESCRIPTION
**Pull request by johnxnguyen**
## What's new in this PR?

This PR includes changes to migrate from Xcode 10.2.3 to Xcode 11.3. Changes include:

- update to recommended project settings

### Needs Release With

- [x] https://github.com/wireapp/wire-ios-shared-resources/pull/39
- [x] https://github.com/wireapp/wire-ios-system/pull/47

### To Do

- [x] Unpin azure pipeline

### Notes

Zero build warnings 🎉

### CODE REVIEW

File: Resources/Configurations/zmc-config/project-common.xcconfig [hunk 1]
```diff
@@ -51,7 +51,7 @@ FRAMEWORK_SEARCH_PATHS = $(CARTHAGE_FOLDER) $(PLATFORM_DIR)/Developer/Library/Fr
 
 // Swift
 //
-SWIFT_VERSION = 5.0
+SWIFT_VERSION = 5.1.3
```
Comment (billypchan) [inline Resources/Configurations/zmc-config/project-common.xcconfig]: Is it necessary to specific patch version?

Comment (johnxnguyen) [reply Resources/Configurations/zmc-config/project-common.xcconfig]: Fixed

File: azure-pipelines.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ resources:
   - repository: wire-ios-shared-resources
     type: github
     name: wireapp/wire-ios-shared-resources
-    ref: refs/heads/master # Branch to fetch the jobs template from
+    ref: refs/heads/chore/xcode11.3 # Branch to fetch the jobs template from
```
Comment (billypchan) [inline azure-pipelines.yml:6]:
```suggestion
    ref: refs/heads/master # Branch to fetch the jobs template from
```

File: Cartfile.resolved [hunk 3]
```diff
@@ -1,2 +1,2 @@
 github "wireapp/ocmock" "v3.4.3"
-github "wireapp/wire-ios-system" "30.0.0"
+github "wireapp/wire-ios-system" "chore/xcode11.3"
```
Comment (billypchan) [inline Cartfile.resolved:2]:
```suggestion
github "wireapp/wire-ios-system" "31.0.0"
```

**Comment by billypchan** [comment]
@zenkin release major

**Comment by billypchan** [comment]
@zenkins release major

**Pull request merged**